src/index.js

```diff
@@ -1,6 +1,27 @@
 import './index.css';
-import getScores from './modules/score.js';
+import displayScores, { scoresTable } from './modules/displayScores.js';
+import getScores from './modules/getScores.js';
+import sendScores from './modules/addScores.js';
 
-const scoresTable = document.querySelector('.scores');
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/aOvKqYy7QCs3ip56vsm2/scores/';
 
-getScores(scoresTable);
\ No newline at end of file
+const addScoreForm = document.getElementById('addScore');
+
+document.querySelector('#refreshBtn').addEventListener('click', () => {
```

Comment:
- [ ] [OPTIONAL] When clicking the refresh button The scores get rendered three times, kindly make sure to fix this bug so your app sends only one request to the server at a time

![Screenshot_20230318_194215](https://user-images.githubusercontent.com/25227022/226129762-c59d4852-2bed-4d9a-9622-59103ff51a2e.png)
![Screenshot_20230318_194729](https://user-images.githubusercontent.com/25227022/226129794-8240a8b5-84ab-4fae-8632-aaaf1ff11695.png)

Reply:
Yes I tried to search for what could have been the problem @Ismailco below are the result but i guess it will be network issues.

There could be several reasons why an API is returning three copies of every data you send to it. Here are some possibilities:

API is returning duplicates: It's possible that the API is intentionally designed to return three copies of the data you send, either as a feature or as a bug. In this case, you should check the API documentation or contact the API provider to see if this behavior is expected.

Network issues: It's possible that the network connection between your application and the API is experiencing some issues that are causing the data to be sent multiple times. This could be due to a slow or unreliable network connection, or issues with the routing of the data packets. To troubleshoot this, you could try sending the data from a different network or using a tool like Wireshark to analyze the network traffic.

Code issue: It's also possible that there is an issue with the code in your application that is causing the data to be sent multiple times. This could be due to a bug in the code, an issue with the way the data is being processed, or a problem with the API client library you are using. To troubleshoot this, you could review your code and check for any potential issues, or try using a different API client library.

In any case, it's important to investigate the root cause of the issue to prevent it from happening again in the future.





Regenerate response

Reply:
I will work on it, thanks a lot @Ismailco